pkg/sink/elasticsearch.go

```diff
@@ -196,8 +196,21 @@ func (e *Elasticsearch) SendMessage(ctx context.Context, event event.Event, even
 }
 
 // SendEvent is no-op.
-func (e *Elasticsearch) SendEvent(_ context.Context, _ any, _ []string) error {
-	return nil
+func (e *Elasticsearch) SendEvent(ctx context.Context, rawData any, sources []string) error {
+	e.log.Debugf(">> Sending to Elasticsearch: %+v", rawData)
+
+	errs := multierror.New()
+	for _, indexCfg := range e.indices {
```

Comment:
This logic is invalid - we send events to all indexes and `sources` are not used. We should do

```go
		if !sliceutil.Intersect(indexCfg.Bindings.Sources, eventSources) {
			continue
		}
```
instead.

Basically please use logic from `SendMessage` in `SendEvent`, and remove `SendMessage`. Thanks!

Reply:
Actually, I removed SendMessage now, and each es flush index is for one source, so I removed source part